src/components/bottomSheet/BottomSheet.tsx

```diff
@@ -1906,12 +1903,13 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
                   accessibilityRole={_providedAccessibilityRole ?? undefined}
                   accessibilityLabel={_providedAccessibilityLabel ?? undefined}
                 >
-                  <DraggableView
+                  <BottomSheetDraggableView
                     key="BottomSheetRootDraggableView"
                     style={contentContainerStyle}
+                    enabled={enableContentPanningGesture}
```

Comment:
i dont think we need to pass it as props, as it is already read internally from `useBottomSheetInternal`

Reply:
This is necessary so that `BottomSheetDraggableView` can also be used when `enableContentPanningGesture` of the `BottomSheet` is set to `false`, see PR description.

I've added an example `DraggableViewExample` which showcases the usage.

Reply:
mmm i see, i think for scalability , i would move the `enableContentPanningGesture` to each scrollable view instead. but we could go ahead with this temporary solution

Reply:
Yeah, in my case I need to wrap an external component so the solution of this MR would work better for me...